dataprofiler/profilers/datetime_column_profile.py

```diff
@@ -127,6 +127,23 @@ def data_type_ratio(self):
             return float(self.match_count) / self.sample_size
         return None
 
+    def diff(self, other_profile, options=None):
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError("Unsupported operand type(s) for diff: '{}' "
+                            "and '{}'".format(cls.__name__,
+                                              other_profile.__class__.__name__))
+
+        # We can use find_diff_of_numbers for max/min to get a timedelta object,
+        # since datetime objects can be compared and subtracted naturally
+        differences = {
+            "min": str(utils.find_diff_of_numbers(self._dt_obj_min, other_profile._dt_obj_min)),
```

Comment:
What does the result look like? Can I see an example?

Reply:
Oh nvm, I see in the test. Was there any options for the output format of the timedelta object?

Reply:
Ya, we should prob use one of the existing formats from the profile

Reply:
There's not a way to directly change the output format as far as I can tell besides writing a custom function. If we were to use one of the existing formats, the only one that seems to be suitable for a difference in times is "%H:%M:%S.%f", but I also don't know how "negative" time differences could be represented in this format and if it's necessarily a better practice if it ends up becoming more unreadable. Thoughts? @gme5078 @JGSweets

Reply:
The only thing I dont like about the current format is that if the data scientists want to set a threshold, it feels like they might have to parse the output. I think we can leave it like this for now and change if our customers ask for it.
We cant really use any of the existing formats in the datetimecolumn since those are mostly dates and dates dont make sense as a metric of difference in time.

Reply:
@gme5078 I guess it depends on the use case. For serialization purposes, the default output for a timedelta object cast to str is the most readable and understandable format to me. However, if diff() is being used in actual code, then maybe we can change this to returning the actual timedelta object. I'm just not sure what the intended use cases of diff() are.